E/Exodus/bundled/patches/0001-Patch-for-changes-to-some-library-names-and-NetCDF-c.patch

```diff
@@ -0,0 +1,73 @@
+From 8047cdbe21a53fa91de8614b4bed4a6c013757b3 Mon Sep 17 00:00:00 2001
+From: cmhamel <cmhamel32@gmail.com>
+Date: Sun, 19 Feb 2023 11:15:34 -0500
+Subject: [PATCH] Patch for changes to some library names and NetCDF cmake
+ build to accomodate for Windows builds with the BinaryBuilder.jl toolset.
+ Four files were changed minimally
+
+---
+ cmake/tribits/common_tpls/find_modules/FindNetCDF.cmake | 6 +++---
+ packages/seacas/applications/epu/EP_Internals.C         | 2 +-
+ packages/seacas/applications/epu/EP_ParallelDisks.C     | 2 +-
+ packages/seacas/libraries/suplib_cpp/sys_info.C         | 2 +-
+ 4 files changed, 6 insertions(+), 6 deletions(-)
+
+diff --git a/cmake/tribits/common_tpls/find_modules/FindNetCDF.cmake b/cmake/tribits/common_tpls/find_modules/FindNetCDF.cmake
+index b3e70fe3cd..2f424bd3f8 100644
+--- a/cmake/tribits/common_tpls/find_modules/FindNetCDF.cmake
++++ b/cmake/tribits/common_tpls/find_modules/FindNetCDF.cmake
+@@ -270,9 +270,9 @@ if ( NetCDF_ROOT OR NetCDF_BIN_DIR )
+         set(NetCDF_NEEDS_PNetCDF "${netCDF_HAS_PNETCDF}")
+     else()
+         # Otherwise, try calling the nc-config shell script
+-        if (WIN32)
+-            message(FATAL_ERROR "nc-config can't be used on Windows, please use CMake to install NetCDF")
+-        endif()
++        #if (WIN32)
++        #    message(FATAL_ERROR "nc-config can't be used on Windows, please use CMake to install NetCDF")
++        #endif()
+         find_program(netcdf_config nc-config
+                        PATHS ${NetCDF_ROOT}/bin ${NetCDF_BIN_DIR}
+ 		           NO_DEFAULT_PATH
+diff --git a/packages/seacas/applications/epu/EP_Internals.C b/packages/seacas/applications/epu/EP_Internals.C
+index 39700e0dcd..0e86f3d7f2 100644
+--- a/packages/seacas/applications/epu/EP_Internals.C
++++ b/packages/seacas/applications/epu/EP_Internals.C
+@@ -23,7 +23,7 @@
+ 
+ #if defined(WIN32) || defined(__WIN32__) || defined(_WIN32) || defined(_MSC_VER) ||                \
+     defined(__MINGW32__) || defined(_WIN64) || defined(__MINGW64__)
+-#include <Shlwapi.h>
```

Comment:
Please upstream the changes in the name of the header files.  This is ***not*** specific to BinaryBuilder, but it's helpful for anyone building on case-sensitive file systems.

Reply:
Ok. I'll make the header file name changes upstream.

Are you ok with me leaving the following part of the patch? Probably not a good idea to make this change upstream.

-        if (WIN32)
-            message(FATAL_ERROR "nc-config can't be used on Windows, please use CMake to install NetCDF")
-        endif()
+        #if (WIN32)
+        #    message(FATAL_ERROR "nc-config can't be used on Windows, please use CMake to install NetCDF")
+        #endif()

Thanks!

Reply:
Yes, although it'd be nice not to have to do that at all eventually.